bin/automerge-prs

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+if [[ -z "${GITHUB_TOKEN:-""}" ]]; then
```

Comment:
I wonder if https://github.com/charmbracelet provides a tool for making env var and param parsing nicer?

Reply:
You're saying what I've got is not "nice"? :wink: 

They've got some good stuff, but I think it's more focused on interactive CLI tools, so I'd be surprised. I can check.

Reply:
I spent too long trying to find a clip of the Derry Girls saying "it's just not as nice" and failed. :laughing:

Reply:
Haha.

Well, if this were actually useful outside of its invocation in the action I would probably add options, and then [reference myself](https://pbrisbin.com/posts/an-opinionated-guide-to-options-parsing-in-shell/) on how to do it nicest.

Reply:
![](https://media.giphy.com/media/l2JJA5fbJ5o328Odi/giphy.gif)